src/core/KeyFacts.js

```diff
@@ -22,8 +22,9 @@ const renderViz = (className) => {
               dichart.showLoading();
               const { keyFacts, location } = window.DIState.getState;
 
-              if (keyFacts && keyFacts.url) {
-                fetchData(keyFacts.url)
+              if (keyFacts && (keyFacts.url || keyFacts.dataID)) {
```

Comment:
Could use optional chaining here? And everywhere else we have `keyFacts.dataID`?